src/App/Action/Plugin/DisablePasswordChangeRequest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Copyright © Vaimo Group. All rights reserved.
+ * See LICENSE for license details.
+ */
+namespace Vaimo\AdminAutoLogin\App\Action\Plugin;
+
+/**
+ * @plugin \Magento\User\Model\Backend\Config\ObserverConfig
+ */
+class DisablePasswordChangeRequest
+{
+
+    /**
```

Comment:
Remove extra line change

```suggestion
{
    /**
```